pages/_app.js

```diff
@@ -1,10 +1,25 @@
 import GlobalStyle from "../styles";
+import useSWR from "swr";
+import { SWRConfig } from "swr";
+import ArtPieces from "@/components/ArtPieces";
+import Navigation from "@/components/Navigation";
+
+const fetcher = (url) => fetch(url).then((res) => res.json());
 
 export default function App({ Component, pageProps }) {
+
+  const { data, error,  isLoading } = useSWR('https://example-apis.vercel.app/api/art', fetcher)
+  const pieces = data
+  console.log("Pieces:", pieces)
+
```

Comment:
Since you're destructuring for `error` and `isLoading` too, it makes sense to add these render conditions too. We can style them later.

```
if (error) return <div>Failed to load</div>;
if (isLoading) return <div>Loading...</div>; 
```